### ISSUE TITLE
Header/footer dev work to add links to new Contact us page and R&S landing page

### ISSUE THREAD

**Issue by bethpottsVADEPO** [department-of-veterans-affairs/va.gov-team#17554]
## Issue Description

We want to add links to the header and footer that go to: 1/ Resources and support landing page and 2/ the new Contact us page. 
**Note:** The new Contact us page isn't launched yet, so we are holding on this ticket until that page to pass an accessibility check. Just wanted to get it into sprint planning for this sprint with the expectation that we'll be able to proceed with this dev work in the next couple days.

---
## Tasks
- [ ] Change the header and footer as shown here:
https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1603977230451/f2be5be57cb4c9ef848586f1ffa821890e3f1544

Summary of what's in the mock-up above:
**Header:** Delete the question mark icon and the caret dropdown. "Contact us" stays there, and instead of being a drop-down, please link those words to: /contact-us

**Footer:** 
**In crisis? Talk to someone now.**
Veterans Crisis Line _(this link stays the same)_

**Get answers**
[Resources and support](/resources)
[Contact us](/contact-us)

**Call us**
800-698-2411 _(keep this number linked as it currently is)_
TTY: 711

**Visit a medical center or regional office**
[Find a VA location](/find-locations)

For the Contact us root-level page, the url is: /contact-us
For the Resources and support link, the url is: /resources

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by jenniferlee-dsva** [comment]
I saw in another footer ticket that we should remove the periods at the ends of those headers...

**Comment by bethpottsVADEPO** [comment]
Changed both--removed the periods and now it's "regional office"

**Comment by brianalloyd** [comment]
@ncksllvn @kelsonic please review and provide estimate, moving into this sprint as this looks like a quick change. Pls advise.

**Comment by kelsonic** [comment]
Picking this up 🙂

**Comment by kelsonic** [comment]
I saw there wasn't a mobile spec, so this is what I have on my current PR:
![image](https://user-images.githubusercontent.com/12773166/103368009-bbfca500-4a83-11eb-98f1-8e0d5c8214f0.png)

This is the desktop version of my PR:
![image](https://user-images.githubusercontent.com/12773166/103368052-d8004680-4a83-11eb-8e21-499342570e86.png)

**Comment by jenniferlee-dsva** [comment]
@kelsonic - @rtwell included mobile version image here: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/15614#issuecomment-737318085

See the section called "Proposed."

**Comment by jenniferlee-dsva** [comment]
Sorry - this is much easier to put it here. The only thing to correct from Ryan's screenshot is omitting the period at the ends of the headings. 

![image.png](https://images.zenhubusercontent.com/5bb2975c143b4d2ee7a35ef6/0c6376b5-eea5-4f30-9ba2-9589bed9fd96)

**Comment by kelsonic** [comment]
Ah perfect! Thanks @jenniferlee-dsva! Do you want me to remove the periods on desktop too?

**Comment by jenniferlee-dsva** [comment]
Also - @kelsonic - per Beth's message above - the text should be - "Visit a medical center or `regional `office" -- not benefit office, per our style guide. (Sorry so many little things.)

**Comment by jenniferlee-dsva** [comment]
>Do you want me to remove the periods on desktop too?

yes, please

**Comment by kelsonic** [comment]
Desktop:
![image](https://user-images.githubusercontent.com/12773166/103369051-6249aa00-4a86-11eb-8b36-552715a32cd6.png)


Mobile:
![image](https://user-images.githubusercontent.com/12773166/103369061-6ece0280-4a86-11eb-8686-f2eb4e77e4ca.png)

**Comment by bethpottsVADEPO** [comment]
@kelsonic These should all be the same font size (the last two look smaller on your desktop screenshot above):
In crisis? Talk to someone now
Get answers
Call us
Visit a medical center or regional office

**Comment by kelsonic** [comment]
Ah good point, I'll see if I can fix that real quick without affecting anything else, the code for it is a bit hard to follow 👍

**Comment by kelsonic** [comment]
Oh where do you want the Contact us to navigate to in the header? 
![image](https://user-images.githubusercontent.com/12773166/103369724-1b5cb400-4a88-11eb-81ed-95ad2b03b7b6.png)

**Comment by kelsonic** [comment]
I'm guessing `tel:18006982411` 🙂

**Comment by kelsonic** [comment]
PR is up: https://github.com/department-of-veterans-affairs/vets-website/pull/15500

**Comment by kelsonic** [comment]
Just saw this comment from @zacharymorel (https://github.com/department-of-veterans-affairs/vets-website/pull/15500#discussion_r550283680), are we wanting to point to the new `/contact-us` page with any of these links? Sorry if I misunderstood! 🙂

**Comment by bethpottsVADEPO** [comment]
sorry to be slow! Yes, we do want the "contact us" link to go to a new page (/contact-us), but it isn't live yet. We're waiting for an accessibility thumbs up. Can you hold on your PR until I get that thumbs up?

For the Contact us root-level page, the url is: /contact-us
For the Resources and support link, the url is: /resources

**Comment by kelsonic** [comment]
Ah perfect! Not a problem, I'll hold off until you ping me with a comment here 🙂 Thanks @bethpotts!

**Comment by bethpottsVADEPO** [comment]
Won't be today, I know that. I just heard from Trevor. He's swamped and had an unexpected OOO. So we'll have to wait. It's possible it'll be tomorrow, but hard to say. Thanks for your patience.

**Comment by bethpottsVADEPO** [comment]
Closing, this work is complete.
